src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/equalshashcode/InputEqualsHashCodeEqualsParameter.java

```diff
@@ -7,7 +7,7 @@
 package com.puppycrawl.tools.checkstyle.checks.coding.equalshashcode;
 
 public class InputEqualsHashCodeEqualsParameter {
-    public static class TestClass1 { // ok
+    public static class TestClass1 {
```

Comment:
The files lgtm....

Squash and rebase to single commit.

Please go through the Issue https://github.com/checkstyle/checkstyle/issues/13213 description before sending PR.
You must also remove the respective suppression from [checkstyle-input-suppressions.xml](https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle-input-suppressions.xml)